File: src/main/java/com/ivaneye/intellijweb2/Main.java
package com.ivaneye.intellijweb2;

import org.mybatis.spring.annotation.MapperScan;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.annotation.ComponentScan;
import org.springframework.transaction.annotation.EnableTransactionManagement;

@SpringBootApplication
@EnableTransactionManagement
@MapperScan({"com.ivaneye.intellijweb2.mapper","com.ivaneye.intellijweb2.respository"})
@ComponentScan({"com.ivaneye.intellijweb2"})
public class Main {

    public static void main(String[] args) throws Exception {
        SpringApplication.run(Main.class, args);
    }
}

File: src/main/kotlin/com/ivaneye/intellijweb2/Main.kt
package com.ivaneye.intellijweb2

import org.mybatis.spring.annotation.MapperScan
import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.context.annotation.ComponentScan
import org.springframework.transaction.annotation.EnableTransactionManagement

/**
 * Created by wangyifan on 18-2-19.
 */
@SpringBootApplication
@EnableTransactionManagement
@MapperScan("com.ivaneye.intellijweb2.mapper", "com.ivaneye.intellijweb2.respository")
@ComponentScan("com.ivaneye.intellijweb2")
open class Application

fun main(args: Array<String>) {
    SpringApplication.run(Application::class.java, *args)
}